templates/master.yaml.tmpl

```diff
@@ -2294,6 +2294,16 @@ storage:
           SUBSYSTEM=="block", KERNEL=="sdc", TAG+="systemd"
           SUBSYSTEM=="block", KERNEL=="sda6", TAG+="systemd"
     {{- end }}
+    - path : /etc/modules-load.d/ipvs.conf
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
+          ip_vs
+          ip_vs_rr
+          ip_vs_wrr
+          ip_vs_sh
+          nf_conntrack_ipv4
```

Comment:
Do we really need `nf_conntrack_ipv4`? it's loaded by default afaik

Reply:
I just took a list of modules from official k8s blog post. Maybe it is not required there, but it is idempotent from the CoreOS version

Reply:
oki :)